packages/typespec-vscode/snippets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Alias": {
+    "prefix": "alias",
+    "body": "alias ${1:AliasName} = ${0:Expression};",
```

Comment:
wonder what's the user experience if it's a multi-line expression, will ENTER just end the snippet so user has to ENTER twice to have a newline? if so, maybe it's better to just put the cursor after '=' instead of having a expression placeholder?

Reply:
its also such a simple snippet this on that we could consider not having it, typescript default snippets don't include one for `type =` either.

Reply:
@RodgeFu Press enter will just create a new line

Reply:
> its also such a simple snippet this on that we could consider not having it, typescript default snippets don't include one for `type =` either.

OK. I deleted `Alias` since it's too simple.